source/Modules/ConfigMgrCBDsc.ResourceHelper/ConfigMgrCBDsc.ResourceHelper.psm1

```diff
@@ -548,6 +548,50 @@ function Convert-CidrToIP
     }
 }
 
+function ConvertTo-CimCMScheduleString
+{
+    [CmdletBinding()]
+    [OutputType([Microsoft.Management.Infrastructure.CimInstance])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [String]
+        $ScheduleString,
+
+        [Parameter(Mandatory = $true)]
+        [String]
+        $CimClassName
+    )
+
+    $schedule = Convert-CMSchedule -ScheduleString $ScheduleString
+
+    if (-not [string]::IsNullOrEmpty($schedule.DaySpan))
+    {
+        if ($schedule.DaySpan -gt 0)
```

Comment:
Can $schedule.DaySpan, HourSpan, or MinuteSPan coexist?

Reply:
you can't mix and match, you can't have hours and minutes, or Days, hours, minutes. it is a chose one type option